### PR TITLE
New publish modal

### PR DESCRIPTION
**Pull request by viddo**
publish-modal2

This implements the new publish modal in the editor.

Solution:
- Basically a new modal for the publish view
- One view/model/template instance per option since sufficiently different 9but the view/model class is reused)
- Reusing styles (OptionCard) from change privacy view

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3390/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3398/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3399/))

**Comment by viddo** [comment]
@xavijam @javisantana please code review, should be straight forward. The only trouble I had was with zclip that I could not really write automated unit tests for, due to how it is works.

cc @saleiva the create-map-first flow is still TBD, also needs to be added to the add-layer-modal so better to solve that flow separately I think. meanwhile re-using the existing modal to create map: 
screen shot 2015-04-30 at 15 44 57

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3401/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3402/))

**Comment by saleiva** [comment]
vamos neno!

**Comment by viddo** [comment]
see ded02 to give it a try

**Comment by viddo** [comment]
friendly bump for code review ;)

**Comment by xavijam** [comment]
Looks good man!

File: app/assets/stylesheets/new_common/option-card.css.scss [hunk 1]
```diff
@@ -24,7 +43,8 @@
   background-color: $cCard-selectedFill;
   cursor: default;
 }
-.OptionCard:not(.is-disabled):not(.is-selected):hover {
+.OptionCard--static { cursor: default }
+.OptionCard:not(.is-disabled):not(.is-selected):not(.OptionCard--static):hover {
```
Comment (xavijam) [inline app/assets/stylesheets/new_common/option-card.css.scss]: :| + :D

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3426/))

**Pull request merged**